ImageNet/inference/AlibabaCloud_resnet26d_1t4_ecs_pytorch_tensorRT.json

```diff
@@ -0,0 +1,18 @@
+{
+    "version": "v1.0",
+    "author": "Perseus AI Cloud Acceleration team in Alibaba Cloud",
+    "authorEmail": "niqi.lnq@alibaba-inc.com, youliang.yl@alibaba-inc.com",
+    "framework": "Pytorch+TensorRT",
+    "codeURL": "https://github.com/ali-perseus/DAWNBench_Inference",
+    "commitHash": "bc904f374c4e6335e964ffeb9a96e5a335471f3b",
+    "model": "ResNet26d",
+    "hardware": "Alibaba Cloud [ecs.gn6i-c8g1.2xlarge]",
+    "latency": 0.637316,
+    "cost": 3.1228484e-07,
+    "top5Accuracy": 93.028,
+    "timestamp": "2019-10-17",
+    "misc": {
+        "TensorRT": "TensorRT 6.0.1.5 for CentOS/RedHat 7 (https://developer.nvidia.com/compute/machine-learning/tensorrt/secure/6.0/GA_6.0.1.5/tars/TensorRT-6.0.1.5.CentOS-7.6.x86_64-gnu.cuda-10.0.cudnn7.6.tar.gz",
```

Comment:
Please add the version of Pytorch.

Reply:
thanks for reply~ 
we update the Pytorch version and torchvision version now!

Reply:
> Please add the version of Pytorch.

thanks for reply~
we update the Pytorch version and torchvision version now!

Reply:
> Please add the version of Pytorch.

we have updated the Pytorch version and torchvision version in new commit!

Reply:
> Please add the version of Pytorch.

Thanks for your reply~ We have updated the Pytorch version and torchvision version in a new commit!